canonical-web-and-design/ubuntu.com
# Updated /legal/terms-of-service markup

## Done

- updated the markup of the page as ol lists were not showing correctly

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/legal/terms-of-service
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the page looks better


## Issue / Card

Fixes #5825
